site/src/hooks-doc/useStep/useStep.demo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { useStep } from 'usehooks-ts'
+
+export default function Component() {
+  const {
+    actualStep,
+    canGoToPrevStep,
+    canGoToNextStep,
+    goToNextStep,
+    goToPrevStep,
+    reset,
+    setStep,
+  } = useStep(5)
```

Comment:
Related to previous comments, if we change the return type and the "actual" word, have to update this too.